homework4.py

```diff
@@ -0,0 +1,32 @@
+# Task description:
+# 1. Створити програму, яка буде очікувати введення тексту від користувача і повідомляти — чи є введений текст “числом” чи “словом”.
+# 2. Якщо введений текст “число”, програма має також вказати чи є воно парним чи непарним.
+# 3. Якщо це “слово”, програма має вказати його довжину.
+
+user_input = input("Enter number or word: ")
+
+if user_input.isdigit():
+    number = int(user_input)
+
+    def is_even():
```

Comment:
I've already seen it somewhere 🤔 
It is not good to copy the solutions of other students. I hope it is just a coincidence. 

Anyway, the comment is the same: 

It is good that you know how to use functions, but in this case, this function makes the code more complicated and less readable and adds one extra line of code.
Functions are good when you do some atomic operation, or when you can reuse it, or to split big part of a program. But in this case, it is unjustified.

Reply:
Thank you for your comment. I just used a function because it was explained during the lecture. I did not copy the code of others :) Please check updated solution without extra line of code.